schema.csv

```diff
@@ -114,13 +114,14 @@ service.state,keyword,0,
 service.type,keyword,0,
 service.version,keyword,0,3.2.4
 source.domain,keyword,0,
+source.full_domain,keyword,0,
 source.hostname,keyword,0,
 source.ip,ip,0,
 source.mac,keyword,0,
 source.port,long,0,
-source.subdomain,keyword,0,
+url.domain,keyword,0,elastic.co
 url.fragment,keyword,0,
-url.host.name,keyword,0,elastic.co
```

Comment:
We can't remove the hostname here but should rename it to `url.hostname`. These entries are pretty standard for url_parsers across various languages.

Reply:
I agree that the field `url.host.name` needs to change to something else. But I disagree that it should become `url.hostname`.

Please read https://github.com/elastic/ecs/issues/84#issuecomment-431951431 to understand why I'd rather introduce `domain` here instead of `hostname`.

Reply:
The url.hostname is what shows up in the url. It could also be an ip address.

Reply:
For the record, @ruflin won me over, at least for `url.hostname`: #147.

The deciding argument was that a URL can either have a domain name or an IP there. So using the word `domain` was not suitable.